.circleci/config.yml

```diff
@@ -309,10 +300,11 @@ jobs:
       - checkout
       - run: git remote add upstream git@github.com:webex/react-widgets.git
       - run: git fetch --all --tags --prune
+      - jq/install
       - run:
           name: 'Grab Version Number'
           command: |
-            PRODUCTION_VERSION_NUMBER=$(curl -f 'https://code.s4d.io/widget-demo/production/.version')
+            PRODUCTION_VERSION_NUMBER=$(curl -s -f 'https://code.s4d.io/widget-space/production/manifest.json' | jq -r '.version')
```

Comment:
```suggestion
            PRODUCTION_VERSION_NUMBER=$(curl -s -f 'https://code.s4d.io/widget-space/production/manifest.json' | jq -r 'manifest.json')
```

Reply:
`.version` is jq syntax to grab the `version` key

Reply:
Ah ok!